docs/Developers/Git_Basics.rst

```diff
@@ -245,6 +245,12 @@ the desired branch:
     $ git fetch origin
     $ git rebase origin/master
 
+.. note::
+
+    ``origin`` is the remote of https://github.com/coala/coala
+    or https://github.com/coala/coala-bears. It is not the remote
+    of your fork.
```

Comment:
I'm moving our discussion out of this review into the main thread for better visibility (this thread would be collapsed with my rebased commit).